### PR TITLE
Add API binding for OPML Export function in the core

### PR DESCRIPTION
**Pull request by Keeper-of-the-Keys**
Needed to allow exporting OPML files from gpodder-sailfish.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -356,6 +364,7 @@ def play_episode(self, episode_id):
         return {
             'title': episode.title,
             'podcast_title': episode.podcast.title,
+            'podcast_coverart': self._get_cover(episode.podcast),
```
Comment (elelay) [inline main.py]: why is this added ?

Comment (Keeper-of-the-Keys) [reply main.py]: I suspect that was added as part of some other work for showing coverart a while ago, I'm puzzled that it isn't in upstream already.

Need to still check what happened there exactly but I also suspect that leaving it is not a bad thing since it is something that is being used afaik.

**Comment by Keeper-of-the-Keys** [comment]
Yeah the coverart thing seems to have bled over from other work.

File: main.py (hunk 1 above)
Comment (Keeper-of-the-Keys) [reply main.py]: After re-checking I at a total loss why that would be considered an addition, it was added in https://github.com/gpodder/gpodder-ui-qml/pull/8/files#diff-5bc02cefb3ea9e27f1a6776eabd1935d

**Comment by Keeper-of-the-Keys** [comment]
As mentioned in the comment on the code itself, that line was already in PR#8 so not sure why it is marked as it is unless I am missing some single character typo...

I think this can be merged if that line is indeed the same as the line from PR#8

File: main.py (hunk 1 above)
Comment (elelay) [reply main.py]: OK. This is because your branch is based on an older version of master, where you don't have the merged #8 code.
